src/js/load-package.ts

```diff
@@ -40,7 +40,7 @@ async function initializePackageIndex(lockFilePromise: Promise<any>) {
   // compute the inverted index for imports to package names
   API._import_name_to_package_name = new Map();
   for (let name of Object.keys(API.lockfile_packages)) {
-    const pkg = API.lockfile_packages[name];
+    const pkg = API.lockfile_packages[name] as InternalPackageData;
```

Comment:
If you give `lockfile_packages` keys type `InternalPackageData` we shouldn't need these casts.